homeassistant/components/media_player/plex.py

```diff
@@ -103,9 +105,14 @@ def setup_plexserver(host, token, hass, config, add_devices_callback):
     """Set up a plexserver based on host parameter."""
     import plexapi.server
     import plexapi.exceptions
+    configurator = get_component('configurator')
 
     try:
         plexserver = plexapi.server.PlexServer('http://%s' % host, token)
+        configurator.request_done('http://' + host)
+        _LOGGER.info("Discovery configuration done (no token needed)")
```

Comment:
Leave this info log output in place so that we have some sort of acknowledgement that Plex was setup without a token being needed.